### DISCUSSION TITLE
[Kanban] Delete card/column confirmation

### DISCUSSION THREAD

**Discussion by AshRubigo** [BurningTreeC/tiddlywiki-muuri#14] [Ideas]
Edit: Added by @BurningTreeC for version 0.1.13.

Currently deleting a card or a column using the kanban interface provides no confirmation, which is in contrast with normal tiddlers. It is very easy to accidently click the delete card button instead of the permalink button of the card beneath in the kanban column.

A confirmation popup should be a settings option and on by default.

**Comment by BurningTreeC** [comment]
Thank you for your feedback @AshRubigo 

I will add a confirmation prompt for deleting cards
